app/src/main/java/com/example/taskwise_eventmaster/presentation/sign_in/SignInViewModel.kt

```diff
@@ -19,17 +18,24 @@ class SignInViewModel @Inject constructor(
     var state by mutableStateOf(SignInState())
         private set
 
-    fun onEvent(event: SignInScreenEvent) {
+    init {
+        if (authService.getSignedInUser() != null) {
```

Comment:
Random question: Is it possible to rewrite this without using an `if` statement, while still preserving the logic?